react/index.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { FC, useState, Fragment } from 'react'
+import { useQuery } from 'react-apollo'
+import { useIntl, defineMessages, FormattedMessage } from 'react-intl'
+import { Helmet, ExtensionPoint, useRuntime } from 'vtex.render-runtime'
+import { Button } from 'vtex.styleguide'
+import { usePWA } from 'vtex.store-resources/PWAContext'
+
+import { OrderGroupContext } from './components/OrderGroupContext'
+import ErrorMessage from './components/ErrorMessage'
+import GET_ORDER_GROUP from './graphql/getOrderGroup.graphql'
+import NotFound from './icons/NotFound'
+import Forbidden from './icons/Forbidden'
+import Skeleton from './Skeleton'
+import { CurrencyContext } from './components/CurrencyContext'
+import { Analytics } from './Analytics'
+
+interface OrderGroupData {
+  orderGroup: OrderGroup
+}
+
+const messages = defineMessages({
+  title: {
+    id: 'store/page.title',
+    defaultMessage: '',
+  },
+  invalidTitle: {
+    id: 'store/order.error.invalid.title',
+    defaultMessage: '',
+  },
+  invalidMessage: {
+    id: 'store/order.error.invalid.message',
+    defaultMessage: '',
+  },
+  notLoggedTitle: {
+    id: 'store/order.error.not-logged-in.title',
+    defaultMessage: '',
+  },
+  notLoggedMessage: {
+    id: 'store/order.error.not-logged-in.message',
+    defaultMessage: '',
+  },
+})
+
+const OrderPlaced: FC = () => {
+  const intl = useIntl()
+  const runtime = useRuntime()
+  const { settings = {} } = usePWA() || {}
```

Comment:
shouldn't it be like this?
```suggestion
  const { settings = {} } = usePWA()
```

Reply:
Nope, because `usePWA` can return `null` and trying to destructure it would result in a `Cannot read property 'settings' of null`